tests/misc/check_init_priorities/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Typo in commit: *exercise
*but that specific bit

Reply:
Sorted, thanks.